cfme/infrastructure/cluster.py

```diff
@@ -117,7 +117,12 @@ class All(CFMENavigateStep):
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
     def step(self):
-        self.parent_view.navigation.select('Compute', 'Infrastructure', 'Clusters')
+        try:
```

Comment:
self.obj.provider raises
AttributeError: type object 'Cluster' has no attribute 'provider'
Don't know how can it be